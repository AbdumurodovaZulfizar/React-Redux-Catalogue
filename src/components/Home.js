import React from 'react';

const Home = () => (
  <div className="home">
    <div className="vawe">
      <div className="row m-0">
        <div className="col-12 col-md-5 font landing-page text-center text-md-start">
          <h1>Cooking with love provides food for the soul</h1>
          <div><button className="btn sign_up__page rounded-pill font px-3" type="submit">Start Cooking</button></div>
        </div>
      </div>
    </div>
  </div>
);

export default Home;
