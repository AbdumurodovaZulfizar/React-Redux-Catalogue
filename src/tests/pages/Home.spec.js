import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../../components/Home';

test('should render home page correctly', () => {
  const component = renderer.create(<Home />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
