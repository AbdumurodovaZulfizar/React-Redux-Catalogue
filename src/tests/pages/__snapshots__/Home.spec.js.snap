// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render home page correctly 1`] = `
<div
  className="home"
>
  <div
    className="vawe"
  >
    <div
      className="row m-0"
    >
      <div
        className="col-12 col-md-5 font landing-page text-center text-md-start"
      >
        <h1>
          Cooking with love provides food for the soul
        </h1>
        <div>
          <button
            className="btn sign_up__page rounded-pill font px-3"
            type="submit"
          >
            Start Cooking
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
